packages/client/README.md

```diff
@@ -634,9 +945,9 @@ await stream.publish(msg, Date.now(), msg.vehicleId)
 
 By default, the JS client subscribes to the first partition (partition `0`) in a stream. The partition number can be explicitly given in the subscribe call:
```

Comment:
This behavior will change in the future so that it will subscribe to all partitions by default.